fix: move a step earlier in the sequence to avoid Gradle sync error (#218)
JetBrains/kotlin-multiplatform-dev-docs: topics/multiplatform-integrate-in-existing-app.md

> You will create a shared module and later connect it to both the existing Android application and your future iOS application:

Reviewer: ```suggestion
Create a shared module and connect it to both the existing Android application and your future iOS application:
```

> 1. In Android Studio settings, turn on the **Enable experimental Multiplatform IDE features** option in the **Advanced Settings** section.

Reviewer: When reproducing, I found it more intuitive to find for the first location (Advanced Settings) first. How about:

In Android Studio settings, select **Advanced Settings**  and turn on the **Enable experimental Multiplatform IDE features** option.

2. Restart Android Studio for the changes to take effect. 

> 3. To avoid classloader issues when Kotlin Multiplatform Gradle Plugin is applied in subprojects, add the following lines

Reviewer: ```suggestion
3. To avoid classloader issues when the Kotlin Multiplatform Gradle plugin is applied in subprojects, add the following lines
```

Reviewer: I found this usage in our docs. Also think it's clearer.

   to the root `build.gradle.kts` file:

    ```kotlin

>     alias(libs.plugins.kotlinMultiplatform) apply false

Reviewer: `alias` is reported as unresolved reference. Is that ok?

Reviewer: If so, maybe we can add a note warning users not to worry about it?

Author: Not really :) if you can, please send your sample to me in DM.

    alias(libs.plugins.androidLibrary) apply false
    ```
   

> 4. Select the **File** | **New** | **New Module** menu item.

Reviewer: ```suggestion
4. Select **File** | **New** | **New Module** from the menu bar.
```

Author: From the IDE docs, I see "in the main menu" as the accepted term, and I do think it's better than both of our alternatives :)

> 5. In the list of templates, select **Kotlin Multiplatform Shared Module**, enter the module name `shared` and the package

Reviewer: ```suggestion
5. In the list of templates, select **Kotlin Multiplatform Shared Module**. Enter the module name `shared` and the package
```

   name `com.jetbrains.simplelogin.shared`.
6. Select **Regular framework** in the **iOS framework distribution** list: this indicates the method you'll use to connect
   the shared module to the iOS application.

   ![Kotlin Multiplatform shared module](multiplatform-mobile-module-wizard.png){width=700}

> 7. Click **Finish**. The wizard will create the Kotlin Multiplatform shared module, update the configuration files, and create sample code

Reviewer: ```suggestion
7. Click **Finish**. The wizard creates the Kotlin Multiplatform shared module, updates the configuration files, and creates sample code
```

Reviewer: Right after clicking "finish", there was a pop-up window with a check list to add files to Git. Do we wanna mention this?

Author: No, I don't think so:
* People have different habits of interacting with Git repos: if you download the code instead of cloning it, it works exactly the same, and there won't be any popup there.
* I assume our audience is familiar enough with Git to not be afraid of a Git dialog :)
